### PR TITLE
Take away xeon_phi_provision hook from non cray platform

### PR DESCRIPTION
**Pull request by riyazhakki**
#### Bug/feature Description
* *PBS is throwing below error message when admin try to import a provisioning hook*
* *qmgr obj=my_provisioning svr=default: Another hook 'PBS_xeon_phi_provision' already has event 'provision', only one 'provision' hook allowed", 'qmgr: hook error returned from server*
* *If there is an issue ID, link to it: [PP-XXXX](https://pbspro.atlassian.net/browse/PP-XXXX)*

#### Affected Platform(s)
* *Non Cray Platforms*

#### Cause / Analysis / Design
* *PBS does not allow to import another provisioning hook since now we have PBS hook called PBS_xeon_phi_provision hook for Cray platform*
* *Above error message is obvious as PBS cannot have more than one provisioning hook*

#### Solution Description
* *We are now taking away PBS_xeon_phi_provision provisioning hook from non Cray platforms thereby PBS will allow admin to import provisioning hook on non-Cray platfroms*

#### Testing logs/output
* *Please attach your test log output from running the test you added (or from existing tests that cover your changes)*

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by riyazhakki** [comment]
[provisioning_ptl_results.txt](https://github.com/PBSPro/pbspro/files/1970969/provisioning_ptl_results.txt)
Attaching existing provisioning PTL test logs

File: src/cmds/scripts/pbs_postinstall.in [hunk 1]
```diff
@@ -969,6 +969,7 @@ create_home
 			xtopview -e "[ -d ${PBS_HOME}/server_priv/ ] && echo ${PBS_LICENSE_INFO} > ${PBS_HOME}/server_priv/PBS_licensing_loc"
 		else
 			[ -d ${PBS_HOME}/server_priv/ ] && echo ${PBS_LICENSE_INFO} > ${PBS_HOME}/server_priv/PBS_licensing_loc
+			rm -f ${PBS_HOME}/server_priv/hooks/PBS_xeon_phi_provision.{HK,PY}
```
[the remark this answers is not in the document]
Comment (riyazhakki) [reply src/cmds/scripts/pbs_postinstall.in]: We can remove the other Cray hooks from non-Cray platform but those are not giving trouble like this provisioning hook.

**Comment by lisa-altair** [comment]
Sometimes people with Cray systems will run the PBS server/sched/comm on a white box instead of the Cray sdb.  Is Xeon Phi provisioning supported in this type of scenario?  If so, it would be bad if the hook got deleted when PBS is installed on a non-Cray node.

File: src/cmds/scripts/pbs_postinstall.in (hunk 1 above)
[the remark this answers is not in the document]
Comment (prakashcv13) [reply src/cmds/scripts/pbs_postinstall.in]: Right, but it does not make sense to have them on a non-Cray machine. However, as the issue at hand is being addressed, I am alright.

**Comment by riyazhakki** [comment]
Done

**Comment by riyazhakki** [comment]
I have now rebased my branch.

**Pull request closed without merging**